src/features/annotation-thread/AnnotationThread.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @flow
+ * @file Annotation Thread Container
+ * @author Box
+ */
+
+import React from 'react';
+import AnnotationThreadContent from './AnnotationThreadContent';
+import API from '../../api/APIFactory';
+import APIContext from '../../elements/common/api-context';
+import Internationalize from '../../elements/common/Internationalize';
+import { DEFAULT_HOSTNAME_API, ORIGIN_ANNOTATION_THREAD } from '../../constants';
+import { withErrorBoundary } from '../../elements/common/error-boundary';
+
+import type APICache from '../../utils/Cache';
+import type { BoxItemPermission, StringMap, Token } from '../../common/types/core';
+import type { ErrorContextProps } from '../../common/types/api';
+
+import './AnnotationThread.scss';
+
+type Props = {
+    annotationId?: string,
+    apiHost?: string,
```

Comment:
I think there's two ways to interpret "top level component":
(1) official BUIE element (i.e. ContentPreview, ContentExplorer, etc...)
(2) the component imported from BUIE into a client application

For (1) we need to add these props of `apiHost` for 3rd party developers to have the flexibility to use, etc... but for (2) especially in the `features` directory it can be anything you want for your 1st party application

If you are using `apiHost`, `requestInterceptor`, etc... then ignore this, but if you're not, then I think we can exclude it